packages/pf-codemods/README.md

```diff
@@ -57,3 +57,21 @@ Out:
 ```jsx
 <ToolbarContent visibility={{ default: "hidden" }} />
 ```
+
+### datalist-remove-ondrags [(#163)](https://github.com/patternfly/pf-codemods/issues/163)
```

Comment:
This should link to the relevant React PR.

Reply:
I'm not seeing it; are we supposed to create them?

https://github.com/patternfly/patternfly-react/issues?q=is%3Aissue+is%3Aopen+DataList

Reply:
Should be this one I believe: https://github.com/patternfly/patternfly-react/pull/8388

Might be because the issue has brackets around "DataList"?

Reply:
oh - i was looking for an issue in pf react for the codemod. i understand what you mean now